src/sentry/uptime/endpoints/validators.py

```diff
@@ -313,3 +356,167 @@ def update(self, instance: Detector, data):
             )
 
         return instance
+
+
+class UptimeMonitorDataSourceValidator(BaseDataSourceValidator[UptimeSubscription]):
+    @property
+    def data_source_type_handler(self) -> type[UptimeSubscriptionDataSourceHandler]:
+        return UptimeSubscriptionDataSourceHandler
+
+    name = serializers.CharField(
+        required=True,
+        max_length=128,
+        help_text="Name of the uptime monitor.",
+    )
+    status = serializers.ChoiceField(
+        choices=list(zip(MONITOR_STATUSES.keys(), MONITOR_STATUSES.keys())),
+        default="active",
+        help_text="Status of the uptime monitor. Disabled uptime monitors will not perform checks and do not count against the uptime monitor quota.",
+    )
```

Comment:
This status actually doesn't control this. This is just used for object lifecycle -- basically for PENDING_DELETION. afaik, would be good to verify.

Reply:
Er, sorry the old `ProjectUptimeSubscription` had this, but the enabled field of the detector controls this now